commands/music/loop.js

```diff
@@ -0,0 +1,53 @@
+/* eslint linebreak-style: 0 */
+const Command = require(`${process.cwd()}/base/Command.js`);
+const { MessageEmbed } = require("discord.js");
+
+class Loop extends Command {
+  constructor(client) {
+    super(client, {
+      name: "loop",
+      description: "This command will display the current playing song.",
+      usage: "np",
+      category: "Music",
+      aliases: ["unloop"],
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+    const voiceChannel = message.member.voiceChannel;
+
+    const noVoiceChannelEmbed = new MessageEmbed()

```

Comment:
This comment also includes lines 24, 29, 34, and 39.

Why create new embeds for different events? Simply missing values during the event. That way you're not creating unnecessary embeds that never get used.